### PR TITLE
Shorten the start validator command and added Thirdweb Guide

### PR DESCRIPTION
**Pull request by Megumiiiiii**
Replacing `<ABSOLUTE_PATH>` with `$(pwd)` so we don't to type the PATH manually

### CODE REVIEW

**Comment by Megumiiiiii** [comment]
Wait, let me added thirdweb guide and fixing all copy button

File: pages/validator.mdx [hunk 1]
```diff
@@ -40,19 +40,15 @@ After cloning the repo, open the `testnet.config` file and replace `<SEPOLIA_RPC
 
 ## Running the Validator
 
-You will need to get the absolute path of your `frame-validator` directory by running:
-```bash filename="" copy
-pwd
-```
-
-You should then be able to start the validator by running:
+- Make sure you are on the `frame-validator` directory.
```
Comment (cygaar) [inline pages/validator.mdx:43]: *in the frame-validator directory

**Comment by cygaar** [comment]
Looks good to me otherwise!

File: pages/validator.mdx [hunk 2]
```diff
@@ -61,6 +57,13 @@ docker run -d \
 docker logs -f frame
 ```
 
+## Oradd `--since` to view shorter logs
```
Comment (cygaar) [inline pages/validator.mdx:60]: * Or add

**Review by cygaar — APPROVED** [review]

**Pull request merged**